docs/samples/topic/authentication.md

```diff
@@ -11,27 +11,128 @@ Default authentication is the most common. Specify this if each user of your pac
 Use system authentication if you as the pack author will provide the necessary tokens to successfully make http requests within your pack. An example would be if your pack returns weather forecasts and the API involved requires an API key, but individual users need not provide their own API key. You as the pack author will register an API key and provide it to Coda, and Coda will apply it to all pack requests regardless of the user.
```

Comment:
Nit, just noticed that the older code uses inconsistent capitalization for pack and http.